pkg/utils/strings.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+// Map applies each of functions to passed slice
+func Map(it []string, fn func(string) string) []string {
```

Comment:
in the future, we might create a common utils package outside operator

Reply:
in the future, Go may add map-reduce functions, but there already libraries like this available.